src/pages/search/registration_search/filters/FacetItem.tsx

```diff
@@ -42,17 +42,15 @@ export const FacetItem = ({ title, children }: FacetItemProps) => {
       <Collapse in={isOpen} timeout="auto" unmountOnExit>
         <List disablePadding>
           {itemsToShow}
-          {!showAll && children.length > itemsToShowByDefault && (
-            <li>
-              <ListItemButton
-                title={t('common.show_more')}
-                dense
-                sx={{ justifyContent: 'space-around' }}
-                onClick={() => setShowAll(true)}>
-                <ExpandMore />
-              </ListItemButton>
-            </li>
-          )}
+          <li>
+            <ListItemButton
+              title={t('common.show_more')}
+              dense
+              sx={{ justifyContent: 'space-around' }}
+              onClick={() => setShowAll(!showAll)}>
+              {showAll ? <ExpandLess titleAccess={t('common.show_fewer')} /> : <ExpandMore />}
```

Comment:
Bør sikkert sjekkes med Axe, men jeg tror `titleAccess={t('common.show_fewer')}` kan fjernes her, og brukes på `title` i stedet 🤔 
```suggestion
              {showAll ? <ExpandLess /> : <ExpandMore />}
```